droidlet/memory/craftassist/mc_memory.py

```diff
@@ -155,11 +155,15 @@ def update(self, perception_output: namedtuple = None, areas_to_perceive: List =
                 for pickable_items in perception_output.agent_pickable_items[
                     "in_perception_items"
                 ]:
-                    self.set_item_stack_position(pickable_items)
+                    if not ItemStackNode.update_item_stack_position(self, pickable_items):
```

Comment:
same here: not clear that `update_item_stack_position` would return anything just from the naming, can we either rename or add a new method to return status ?